<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tic_tac_toe</class>
 <widget class="QWidget" name="Tic_tac_toe">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>tic-tac-toe</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Без названия.png</normaloff>Без названия.png</iconset>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn9">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn7">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn4">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn8">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn6">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn5">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset-game</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gubbi</family>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
